cmd/podman/shared/parse/parse.go

```diff
@@ -162,3 +163,9 @@ func ValidateFileName(filename string) error {
 	}
 	return nil
 }
+
+// ValidURL checks a string urlStr is a url or not
+func ValidURL(urlStr string) error {
+	_, err := url.ParseRequestURI(urlStr)
+	return errors.Wrapf(err, "invalid url path: %q", urlStr)
```

Comment:
it seems to me that doing this wrapping mean ValidURL always returns error, regardless of if the url is valid?

Reply:
yes, I forget to change when I wrap the error

Reply:
but yes passing a nil err to wrapf return nil